concurrentTextEditor/dialogsignup.cpp

```diff
@@ -48,16 +48,44 @@ void dialogsignup::on_pushButton_Signup_clicked()
     signup["email"] = email;
     if(this->icn.isNull() || this->icn.isEmpty())
         signup["icon"] = this->_defaultIcon;

```

Comment:
_defaultIcon da prendere dal server così togliamo i path delle icone dal client
in teoria basta una query sul database che ha il path dell'icona